sbp_register.py

```diff
@@ -65,8 +91,7 @@ def on_change(self, register):
             return
 
         # Get the region
-        sbp_point_register.store(register, (self.view, self.view.window(), sel[0]))
-
+        sbp_point_registers.store(register, (self.view, self.view.window(), sel[0], self.view.substr(self.jove.get_encompassing_region())[0:256]))
```

Comment:
Here instead of getting the whole region. I actually just want to get the first 256 characters from the line the cursor is on or to the end of the line if it is less than 256 characters.